incubator/vertx/Makefile

```diff
@@ -0,0 +1,40 @@
+init-image:
+	docker build -f Dockerfile.init -t oscardovao/vertx-init:1.8 .
```

Comment:
I see that there are several files from the `java` runtime in this folder. To avoid having the same files, would you mind modifying this `make` task to copy on the fly the needed files here before building the image? That way is much easier to detect what's different from the mainstream java image.

Reply:
I have modified make to copy duplicated files from the stable java folder